### DISCUSSION TITLE
gelogde duiken nummeren

### DISCUSSION THREAD

**Discussion by easy-diver** [Nelos-VZW/dives-2#113] [Ideas]
is het een optie om je inloggende duiken een nummer toe te kennen die overeenstemt met je eigen logboek waarvan een groot deel niet beschikbaar is in dives.
zo kan je zelf je duik/volgnummer ingeven.

**Comment by ndsmyter** [comment]
Zie https://github.com/Nelos-VZW/dives-2/issues/349#issuecomment-1635483023
